### DISCUSSION TITLE
Trouble getting https working

### DISCUSSION THREAD

**Discussion by calebncontreras** [leokhoa/laragon#468] [Q&A]
Hi, so I activated SSL on Apache2 and I'm seeing the green padlock next to it in the active device list, but when I try to open one of my local sites it does not load with https. Im assuming that laragon is trying to load the sites on port 80 instead of 443. Do I have to explicitly tell Laragon to open my sites on port 443 or will it just default to 443 if ssl is activated?

Oh, looks like I just needed to restart the server lol

**Comment by Pen-y-Fan** [comment]
See [How to set up Laragon on a new Windows computer (part 4) - Enable security](https://pen-y-fan.github.io/2022/11/10/how-to-set-up-laragon-on-a-new-windows-computer-part-4/) for more information.
